core/src/tower_storage.rs

```diff
@@ -248,8 +308,9 @@ impl TowerStorage for EtcdTowerStorage {
         )))
     }
 
-    fn store(&self, saved_tower: &SavedTower) -> Result<()> {
-        let (instance_key, tower_key) = Self::get_keys(&saved_tower.node_pubkey);
+    // TODO: make compatible
```

Comment:
Is this TODO still needed?

Reply:
good catch this isn't needed.

Reply:
Instead it should be TODO: check if this works 😛

Reply:
I've only checked with the FileTowerStorage, do you know how I could setup a local/gce cluster with etcd instead?